### ISSUE TITLE
NPCs aggro through solid walls.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#1076]
This is something new, with the latest patches. NPCs will aggro through solid walls now, without line of sight check.

**Comment by thepparker** [comment]
Can you give an example

**Comment by Cabro** [comment]
Sure. The pirates in Tanaris boats get aggroed from 1 ship through to another, in STV near Mogh the Undying there are some walls through which the adds aggro you without being in line of sight, 2 examples that come to mind.
PS: Merry Christmas!

**Comment by ifyousayso** [comment]
Well, almost all walls in STV are "transparent".. shouldn't be but have always been on private realms. It's extremely difficult to move around in the ruins without pulling something.

The ships used to work properly.. except that Blizzard pulled from the floors below and above, but that's a.. hmm.. perhaps the same issue.

**Comment by Cabro** [comment]
Yes i noticed it mostly around the STV ruins walls, as you mention.
